client/src/app/shared/guard/role.guard.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
+import { Observable } from 'rxjs';
+import { AppStateService } from '../service/app-state.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RoleGuard implements CanActivate {
```

Comment:
Ez a guard nem Role-t nez, hanem bejelentkezest, ugye? H be van-e jelentkezve a user. Ezesetben a neve felrevezeto. LoggedinGuard jobb lenne.

Reply:
kész